graderutils/tracebackformat.py

```diff
@@ -112,13 +115,20 @@ def clean_feedback(result_groups, config):
         for result in group["testResults"]:
             # Run all cleaning tasks for traceback
             for task in config:
-                if task.get("hide_tracebacks", False):
+                if task.get("hide_tracebacks", False) or task.get("remove_sentinel", ''):
```

Comment:
It would be better to call `.get` only once here. Basically:

```python
hide_tracebacks = task.get('hide_tracebacks', False)
remove_sentinel = task.get('remove_sentinel'. '')
if hide_tracebacks or remove_sentinel:
    ...
```